apps/judicial-system/backend/src/app/messages/notifications.ts

```diff
@@ -122,19 +122,27 @@ export const notifications = {
   defenderResubmittedToCourt: defineMessages({
     subject: {
       id:
-        'judicial.system.backend:notifications.defender_resubmitted_to_court.subject_v2',
-      defaultMessage: 'Krafa um {caseType} send aftur',
+        'judicial.system.backend:notifications.defender_resubmitted_to_court.subject',
```

Comment:
Should this be a new id? You mentioned that strings are not removed from contentful and we need to make new ids each time.
```suggestion
        'judicial.system.backend:notifications.defender_resubmitted_to_court.subject_v3',
```

Reply:
Already fixed